app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>YourFinance</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+    <div class="card d-flex align-self-center border-0" >
+      <% if user_signed_in?%>
+        <%= button_to 'Sign Out', "/users/sign_out", method: :delete %>
+      <% end %>
+      <div class="card-body align-self-center border-3 border border-light rounded-3 my-3 pb-auto shadow-lg container-width" style="max-width: 575px">
+        <nav class="navbar navbar-dark bg-primary bg-gradient shadow">
+          <div class="container-fluid d-flex justify-content-space-between">
+          <%= link_to '<', "/", class: 'fs-5 navbar-toggler text-decoration-none' %>
+            <a class="navbar-brand fs-1" href="#"><%= yield :title %></a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+        </nav>
+        <%= yield %>
+      </div>
+    </div>
```

Comment:
- [ ] The layout should be consistent with the original design layout. Kindly compare the attached screenshot below and make your design as close as the original one. :pray: 

![image](https://user-images.githubusercontent.com/64894927/144658739-0bfa47ed-55d1-4f39-8fcc-8d8b40077cac.png)

![image](https://user-images.githubusercontent.com/64894927/144658851-e8790861-0c51-4ef7-88d5-0b45844aa937.png)

Reply:
Rails is not meant to build mobile applications so that's why I made adjustments in the layout to see the "mobile view " with the full display of the screen